orbitx/flight_gui.py

```diff
@@ -32,73 +34,37 @@
 class FlightGui:
 
     def __init__(self, physical_state_to_draw: protos.PhysicalState, texture_path: str =None, no_intro: bool =False )-> None: 
-        ctrl_c_handler: function = signal.getsignal(signal.SIGINT)
         # Note that this might actually start an HTTP server!
-        #import vpython
+        assert len(physical_state_to_draw.entities) >= 1
 
         ################################ Attributes ############################### 
 
-        self._vpython: module = vpython        
-
-        if self._vpython.__version__ == '7.4.7':
-            # vpython installs "os._exit(0)" signal handlers in 7.4.7.
-            # Very not good for us.
-            # Reset the signal handler to default behaviour.
-            signal.signal(signal.SIGINT, ctrl_c_handler)
-
-            # os._exit(0) is in one other place, let's take that out too.
-            # Unfortunately, now that vpython isn't calling os._exit, we have
-            # to notify someone when the GUI closed.
-            # Note to access vpython double underscore variables, we have to
-            # bypass normal name mangling by using getattr.
-            def callback(*_):
-                getattr(self._vpython.no_notebook, '__interact_loop').stop()
-                self.closed = True
-            self.closed = False
-            getattr(self._vpython.no_notebook,
-                    '__factory').protocol.onClose = callback
-
-        # Set up our vpython canvas and other internal variables
-        self._scene = vpython.canvas(
-            title='<b>OrbitX</b>',
-            align='right',
-            width=800,
-            height=600,
-            center=vpython.vector(0, 0, 0),
-            up=vpython.vector(0, 0, 1),
-            forward=vpython.vector(0.1, 0.1, -1),
-            autoscale=True
-        )
-
+        self._last_physical_state: protos.PhysicalState = physical_state_to_draw
+        # self._vpython: module = vpython
+        self._vpython = vp.vp()        
```

Comment:
we shouldn't have to keep the self._vpython hack around, just typing `vpython.sphere()` is much cleaner than `self._vpython.sphere()`